notebooks/official/experiments/get_started_with_comparing_local_trained_models.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<p>new version of the cell:</p><p> </p><p># If you are running this notebook in Colab, run this cell and follow the</p><p># instructions to authenticate your GCP account. This provides access to your</p><p># Cloud Storage bucket and lets you submit training jobs and prediction</p><p># requests.</p><p> </p><p>import os</p><p>import sys</p><p> </p><p># If on Vertex AI Workbench, then don't execute this code</p><p>IS_COLAB = False</p><p>if not os.path.exists("/opt/deeplearning/metadata/env_version") and not os.getenv(</p><p>&nbsp;&nbsp;"DL_ANACONDA_HOME"</p><p>):</p><p>&nbsp;&nbsp;if "google.colab" in sys.modules:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;IS_COLAB = True</p><p>&nbsp;&nbsp;&nbsp;&nbsp;from google.colab import auth as google_auth</p><p> </p><p>&nbsp;&nbsp;&nbsp;&nbsp;google_auth.authenticate_user()</p><p> </p><p>&nbsp;&nbsp;# If you are running this notebook locally, replace the string below with the</p><p>&nbsp;&nbsp;# path to your service account key and run this cell to authenticate your GCP</p><p>&nbsp;&nbsp;# account.</p><p>&nbsp;&nbsp;elif not os.getenv("IS_TESTING"):</p><p>&nbsp;&nbsp;&nbsp;&nbsp;%env GOOGLE_APPLICATION_CREDENTIALS ''</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>done.</p>